### ISSUE TITLE
OSD volume-change notifications stopped appearing

### ISSUE THREAD

**Issue by ecc1** [stefansundin/powermate-linux#5]
Not sure which component is at fault, but turning the powermate knob now changes the volume, but doesn't trigger any OSD display.  It used to.

I'm using XFCE, and I have the pulseaudio plugin in my panel.  When I mouse over it, it displays the volume (in percent), and if I turn the powermate that percentage updates correctly.  If I use the scroll up/down wheel on my mouse while hovering over that volume control, it shows the OSD that I expect from powermate.

I don't see any code in powermate about notification, so I suspect this is a pulseaudio/xfce issue.  Posting just in case you've already encountered this.

**Comment by stefansundin** [comment]
You are right that there's no code to display the notification. Another user submitted a PR to manually show a notification: https://github.com/stefansundin/powermate-linux/pull/1

I am using regular Ubuntu 17.04 and I am getting a notification. I'm not sure exactly what is responsible for making it work though.

**Comment by mrjones-plip** [comment]
@ecc1 - Not sure if you're still looking for fix to this 2+ years later, but a workaround is to use the [config file](https://github.com/stefansundin/powermate-linux/blob/master/powermate.toml) and then specify the the `xdotool` commands cited in the comments.  For me, on Ubuntu 18.04, this triggers the OSD as expected. I'm not sure if XFCE vs Gnome will cause this to not work for you though.

For me, this is a much easier fix than having to recompile off the  branch in #1 ;)
